### PR TITLE
Infra as code staging

### PR DESCRIPTION
**Pull request by kammerdiener**

### CODE REVIEW

File: app/retail/templates/shared/activity.html [hunk 1]
```diff
@@ -486,7 +486,9 @@
     </div>
     {% if row.metadata.resource.type == 'background' and not for_email %}
       <div class="status_background_container">
-        <img class="status_background" src="/static/status_backgrounds/{{row.metadata.resource.id}}.png" />
+          {% with 'status_backgrounds/'|add:row.metadata.resource.id|add:'.png' as background_file %}
```
Comment (gdixon) [inline app/retail/templates/shared/activity.html:489]: nitpick, this needs reindenting

File: .github/workflows/ci-stage.yml [hunk 2]
```diff
@@ -0,0 +1,224 @@
+name: Build & deploy to staging
+
+on:
+  # run it during push to branch
+  push:
+    branches: [ infra_as_code_staging ]
```
Comment (kevinrolsen) [inline .github/workflows/ci-stage.yml:6]: Will we be tracking pushes to this branch going forward to push code to staging.gitcoin.co

**Review by gdixon — APPROVED** [review]
Code changes lgtm! 🚀

File: .github/workflows/ci-stage.yml (hunk 2 above)
Comment (kammerdiener) [reply .github/workflows/ci-stage.yml:6]: We will! Appreciate you catching this!

File: .github/workflows/ci-stage.yml [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+      # - name: Generate Markdown documentation and static docs page
+      #   run: pydocmd build
+
+      # - name: Deploy to Github Pages 🚀
+      #   uses: peaceiris/actions-gh-pages@v3
+      #   if: github.ref == 'refs/heads/master'
+      #   with:
+      #     github_token: ${{ secrets.GITHUB_TOKEN }}
+      #     publish_dir: _build/site
+      #     cname: docs.gitcoin.coind
```
[the remark this answers is not in the document]
Comment (kammerdiener) [reply .github/workflows/ci-stage.yml:116]: Currently no this should not be needed anymore.

**Review by kevinrolsen — APPROVED** [review]
lgtm

**Pull request merged**